python/PEDvPUDplot.py

```diff
@@ -108,6 +170,54 @@ def getPED(filepath2):
             PEDS[key] = value
     return PEDS
 
-PUD = getPUD(filepath)
-PED = getPED(filepath2)
+def PUDxPEDy(filepath, filepath2):
+    """
+    Calls the functions getPUD and getPED and returns distance values for corresponding keys in both dictionaries, as two lists packed in a tuple.
+    
+    ::param filepath::  Path to SDT file (input).
+    ::param filepath2:: Path to Tree-Puzzle file (input).
+    ::param PUDdict::   Dictionary of pairwise sequence keys and uncorrected distance values, excluding pairwise comparisons of sequence with itself.
+    ::param PEDdict::   Dictionary of pairwise sequence keys and evolutionary distance values, excluding pairwise comparisons of sequence with itself.
+    ::param x::         List of evolutionary distances.
+    ::param y::         List of uncorrected distances.
+    """
+    PUDdict = getPUD(filepath)
+    PEDdict = getPED(filepath2)
+    x = []
+    y = []
+    for key in PUDdict.keys():
+        y.append(PUDdict[key])
+        x.append(PEDdict[key])
+    return (x,y)
+
+def reportDistances(filepath, filepath2):
+    """
+    Prints sequence pairs and distances (uncorrected and evolutionary) to file.
+    
+    ::param filepath::  Path to SDT file (input).
+    ::param filepath2:: Path to Tree-Puzzle file (input).
+    ::param PUDdict::   Dictionary of pairwise sequence keys and uncorrected distance values, excluding pairwise comparisons of sequence with itself.
+    ::param PEDdict::   Dictionary of pairwise sequence keys and evolutionary distance values, excluding pairwise comparisons of sequence with itself.
+    ::param report::    Function output written to file.        
+    """
+    PUDdict = getPUD(filepath)
+    PEDdict = getPED(filepath2)
+    with open("PUDPED.txt", "w") as report:
+        report.write("sequencePair, uncorrectedDistance, evolutionaryDistance \n")
+        for key in PUDdict.keys():
+            report.write(str(key) + ", " + str(PUDdict[key]) + ", " + str(PEDdict[key]) + "\n")
+    return report
+
+# Output pairwise distances
```

Comment:
I realized later that I should've just written this as one script rather than trying to make separate functions, since I need this for the specific purpose of working with these specific files.